SPECS/metal3-cluster-api-provider-metal3/metal3-cluster-api-provider-metal3.spec

```diff
@@ -0,0 +1,64 @@
+Summary:        Cluster API Provider Metal3 enables users to deploy a Cluster API based cluster on top of bare metal infrastructure using Metal3.
+Name:           metal3-cluster-api-provider-metal3
+Version:        0.4.0
```

Comment:
do you need this specific version ? github repo is at 0.5.4. If you don't need this specific version I strongly recommend you upgrade to latest. We're currently in the process of upgrading all spec in main to their latest...